vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/DatabaseBackedProcessor.java

```diff
@@ -46,6 +47,14 @@
 
 public abstract class DatabaseBackedProcessor
     extends BaseProcessor<DatabaseWorkflow, DatabaseOperation, DSLContext> {
+
+  private static final Counter badRecoveryCount =
+      Counter.build(
+              "vidarr_db_processor_recovery_failure_count",
+              "The number of failures in recovering database-backed operations")
+          .labelNames("workflow")
```

Comment:
For clarity, could this be changed to `workflow_run`?

Reply:
If we decide to keep this metric label, sure